src/item.cpp

```diff
@@ -1017,21 +1016,16 @@ std::string item::info( bool showtext, std::vector<iteminfo> &info ) const
             info.emplace_back( "GUN", _( "Sight dispersion: " ), "", eff_disp, true, "", true, true );
         }
 
-        info.push_back( iteminfo( "GUN", _( "Recoil: " ), "", mod->gun_recoil( false ), true, "", false,
-                                  true ) );
-        if( has_ammo ) {
-            temp1.str( "" );
-            temp1 << ( ammo_recoil >= 0 ? "+" : "" );
-            // ammo_recoil and sum_of_recoil don't need to translate.
-            info.push_back( iteminfo( "GUN", "ammo_recoil", "",
-                                      ammo_recoil, true, temp1.str(), false, true, false ) );
-            info.push_back( iteminfo( "GUN", "sum_of_recoil", _( " = <num>" ),
-                                      mod->gun_recoil( true ), true, "", false, true, false ) );
+        info.emplace_back( "GUN", _( "Handling: " ), "", mod->gun_handling(), true, "", mod->gun_recoil() == 0 );
```

Comment:
Is this "human-readable"? Can a newb who doesn't know inner workings of recoil system understand this value? If not, is there a way to display it that would allow a newb to get some info out of it?
For example, "Recoil drop at 150/300/600: 10/5/3"